barretenberg/cpp/src/barretenberg/client_ivc/mock_kernel_pinning.test.cpp

```diff
@@ -33,7 +33,8 @@ TEST_F(MockKernelTest, PinFoldingKernelSizes)
         Builder circuit = circuit_producer.create_next_circuit(ivc);
 
         ivc.accumulate(circuit);
+        EXPECT_FALSE(circuit.blocks.has_overflow); // trace oveflow mechanism should not be triggered
```

Comment:
noticed this could silently overflow the trace while still not crossing the specified dyadic size